Calculator/Calculator/ExpressionParser.swift

```diff
@@ -0,0 +1,23 @@
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        var operands = CalculatorItemQueue<Double>()
+        var operators = CalculatorItemQueue<Operator>()
+        
+        input.compactMap { Operator(rawValue: $0) }.forEach { operators.enqueue(element: $0) }
+        componentsByOperators(from: input).compactMap { Double($0) }.forEach { operands.enqueue(element: $0) }
+        
+        return Formula(operands: operands , operators: operators )
+        
+    }
+    
+    private static func componentsByOperators(from input: String) -> [String] {
+        var result: Array<String> = [input]
+        
+        Operator.allCases.forEach { Operator in
+            result = result.map{ $0.split(with: Operator.rawValue) }.flatMap { $0 }
+        }
```

Comment:
`Operator` 인수명은 소문자로 시작합니다! `operator`와 같이 예약어를 사용하는게 권장되는 방법은 아니라 다른 표현을 사용하셔도 좋습니다.
```suggestion
        Operator.allCases.forEach { `operator` in
            result = result.map{ $0.split(with: `operator`.rawValue) }.flatMap { $0 }
        }
```

Reply:
operatorElement로 수정하였습니다